v2/emailpassword/nextjs/setting-up-frontend.mdx

```diff
@@ -30,19 +34,19 @@ import AppInfoForm from "/src/components/appInfoForm"
 ```tsx title="pages/auth/[[...path]].tsx"
 import React, { useEffect } from 'react'
 import dynamic from 'next/dynamic'
-import SuperTokens from 'supertokens-auth-react'
+import { ^{recipePreBuiltUINameCapitalLetters} } from "supertokens-auth-react/recipe/^{codeImportRecipeName}/preBuiltUI";
 import { redirectToAuth } from 'supertokens-auth-react'
 
-const SuperTokensComponentNoSSR = dynamic<React.ComponentProps<typeof SuperTokens.getRoutingComponent>>(
-  new Promise((res) => res(SuperTokens.getRoutingComponent)),
+const SuperTokensComponentNoSSR = dynamic<React.ComponentProps<typeof ^{recipePreBuiltUINameCapitalLetters}.getRoutingComponent>>(
+  new Promise((res) => res(^{recipePreBuiltUINameCapitalLetters}.getRoutingComponent)),
```

Comment:
this will also have to change to account for other recipes being used - like email verification.. how will this code snippet change? And why didn't you think of this on your own - please focus on developer experience.

Reply:
The NextJs example only uses a single recipe. 
It changes by having multiple of these `NoSSR` components (at least, that's the easiest way). Check [here](https://github.com/supertokens/supertokens-auth-react/blob/feat/duplicate-init-split-prebuilt-ui/examples/with-next-iframe/pages/auth/%5B%5B...path%5D%5D.js)
One thing we could change is to include the recipe name in the `NoSSR` component name.